src/cli.rs

```diff
@@ -231,6 +231,18 @@ pub enum EnvCmd {
     /// Delete a environment
     #[command(name = "rm", alias = "remove")]
     Rm(action::env::RmArgs),
+    Header {
+        #[command(subcommand)]
+        command: HeaderEnvCmd,
+    },
+}
+
+#[derive(Debug, Subcommand)]
+pub enum HeaderEnvCmd {
+    Set { headers: Vec<String> },
+    Ls,
+    Rm(action::env::HeaderRmArgs),
+    Get(action::env::HeaderGetArgs),
```

Comment:
Could we use the same Cmd struct as for endpoint's Header? That would be useful to preserve documentation and the same usage.
The same is true for the `Headers` struct. Having 2 distinct implementations (`endpoint::Headers` and `env::Headers`) is kinda confusing, as they should both implement the same functionality.

This also avoids us needing to reimplementing all the traits.

Reply:
Yeah, that makes sense!